tests/IntegrationTests/TokenAcquirerTests/TokenAcquirer.cs

```diff
@@ -287,12 +287,10 @@ private static async Task CreateGraphClientAndAssert(TokenAcquirerFactory tokenA
             var serviceProvider = tokenAcquirerFactory.Build();
             GraphServiceClient graphServiceClient = serviceProvider.GetRequiredService<GraphServiceClient>();
             var users = await graphServiceClient.Users
-                .Request()
-                .WithAppOnly()
-                .WithAuthenticationScheme(s_optionName)
-                //     .WithAuthenticationOptions(options => options.ProtocolScheme = "Pop")
-                .GetAsync();
-            Assert.True(users.Count >= 56);
+                .GetAsync(o => o.Options
```

Comment:
This needed to change as TokenAcquirerTests need Graph 5X for the CertificateRotationTests, and therefore
this test CreateGraphClientAndAssert now leverages Microsoft.Identity.Web.GraphServiceClient and no longer
Microsoft.Identity.web.MicrosoftGraph